catalog_tools/plots/basics.py

```diff
@@ -1,14 +1,34 @@
 import numpy as np
 import matplotlib.pyplot as plt
-from typing import Union
+from typing import Optional
 
 # Own functions
 from catalog_tools.utils.binning import bin_to_precision
 
 
-def plot_cum_fmd(ax: plt.Axes, mags: np.ndarray, color: str = 'blue',
-                 b_value: Union[float, None] = None,
-                 mc: Union[float, None] = None, delta_m: float = 0):
+def gutenberg_richter(magnitudes: np.ndarray, b_value: float,
+                      mc: float, n_mc: int) -> np.ndarray:
+    """ gives the number of the cumulative gutenberg-richter law for a given
```

Comment:
this description was a bit confusing at first. I thought it is about observed magnitudes. maybe mention that this is the theoretical complementary cumulative FMD.